docs/source/deployment/index.rst

```diff
@@ -124,24 +124,19 @@ MLflow offers support for a variety of deployment targets. For detailed informat
                 </a>
             </div>
             <div class="simple-card">
-                <a href="deploy-model-to-ray-serve.html">
+                <a href="../plugins.html#deployment-plugins">
```

Comment:
Ray serve plugin is not actively maintained. We will probably replace this with deployment guide for LLMs, but for the time being just link to community plugins.